nsv13/code/controllers/subsystem/ranks.dm

```diff
@@ -0,0 +1,59 @@
+//For code/controllers/subsystem/job.dm
+/datum/controller/subsystem/job/proc/LoadRanks(rankfile="config/ranks/military.txt")
+	if (fexists("[rankfile]"))
+		var/rankstext = file2text("[rankfile]")
+		for(var/datum/job/J in occupations)
+			var/regex/jobs = new("[J.title]=(.+)")
+			jobs.Find(rankstext)
```

Comment:
Test to see what happens if a job doesnt have a rank and what happens ingame when that occurs. See about adding a alert that is visible in world when that happens while this is loading so everyone can notice and report it to get it fixed.

Reply:
All other jobs are updated and the missing job retains its old rank. What do you think is desirable behavior - default to no rank, or abort the change entirely?

Reply:
Ask Kmc.